plugins/org.eclipse.fordiac.ide.product/org.eclipse.fordiac.ide.product.target

```diff
@@ -59,13 +59,17 @@
 			<unit id="org.eclipse.sirius.samples.feature.group" version="7.4.0.202403051613"/>
 			<unit id="org.eclipse.sirius.specifier.feature.group" version="7.4.0.202403051613"/>
 			<unit id="org.eclipse.sirius.specifier.properties.feature.feature.group" version="7.4.0.202403051613"/>
-			<unit id="org.eclipse.swtbot.eclipse.feature.group" version="4.2.0.202405291350"/>
-			<unit id="org.eclipse.swtbot.eclipse.gef.feature.group" version="4.2.0.202405291350"/>
-			<unit id="org.eclipse.swtbot.feature.group" version="4.2.0.202405291350"/>
-			<unit id="org.eclipse.swtbot.ide.feature.group" version="4.2.0.202405291350"/>
-			<unit id="org.eclipse.swtbot.generator.feature.feature.group" version="4.2.0.202405291350"/>
 			<unit id="org.eclipse.xtext.sdk.feature.group" version="2.35.0.v20240528-1523"/>
 		</location>
+		<location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
+      		<unit id="org.eclipse.swtbot.eclipse.feature.group" version="4.2.1.202406141605"/>
+      		<unit id="org.eclipse.swtbot.eclipse.gef.feature.group" version="4.2.1.202406141605"/>
+      		<unit id="org.eclipse.swtbot.feature.group" version="4.2.1.202406141605"/>
+      		<unit id="org.eclipse.swtbot.generator.feature.feature.group" version="4.2.1.202406141605"/>
+      		<unit id="org.eclipse.swtbot.ide.feature.group" version="4.2.1.202406141605"/>
+      		<unit id="org.eclipse.swtbot.nebula.nattable.finder" version="4.2.1.202406141605"/>
+      		<repository id="SWTBot-3_1" location="https://download.eclipse.org/technology/swtbot/releases/4.2.1/"/>
+    	</location>
```

Comment:
I've made changes to update the SWTBot plugins to version 4.2.1. This change was necessary to meet our requirement for using nattable.finder. Based on the recommendation from the SWTBot developers, we need to use version 4.2.1. Therefore, I've separated the SWTBot plugins into their own repository, as the current repository does not include the 4.2.1 version plugins.